components/NewComment.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { postNewComment } from "../src/api";
+
+function NewComment({setComments}) {
+    const [input, setInput] = useState("")
+    const [err,setErr] = useState(null)
+    const handleChange = (event) => {
+        setInput(event.target.value)
+    }
+    const {article_id} = useParams();
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        postNewComment(article_id,"tickle122",input)
+        .then((newCommentsFromApi) => {
+            setInput("")
+            setComments((currComments) => {
+                return[newCommentsFromApi,...currComments]
+            })
+        })
+        .catch((err) => {
+            setErr('Something went wrong, please try again.')
+        })
+        setInput("")
+    }
+
+
+
+    return (
+  
+        <form className="new-comment">
+            <label> New comment:   
+                <input className="new-comment" type="text" placeholder = "Please input your comment here" onChange ={handleChange} value ={input}/> 
```

Comment:
This would be better if you changed it to a text area to support multi line comments.